plugins/plugin-chart-table/types/external.d.ts

```diff
@@ -1 +1,2 @@
 declare module '*.png';
+declare module 'regenerator-runtime/runtime';
```

Comment:
are there no types available for this?

Reply:
There is no types and it also doesn't need it since it just injects a runtime function to globals.